### PR TITLE
Adding 2 examples - components1.js and components_animate.js

### PR DESCRIPTION
**Pull request by diatrophy**
These examples showcase the use of the pxComponents framework that is a wrapper for the pxscene functionality.
- **_components1.js**_ : This draws 6 images each with different combinations of effects applied - _polaroid, drop shadow and top shadow_
- **_components_animate.js**_ : This draws a single image outside the viewing area, and animates it so that it flys into the viewing area (with the polaroid effect applied)

The pxComponents framework is currently hosted at https://github.com/diatrophy/pxComponents

### CODE REVIEW

File: examples/px-reference/gallery/components1.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+var remote    = "https://diatrophy.github.io/pxComponents/"
```
Comment (conniefry) [inline examples/px-reference/gallery/components1.js]: I notice that when I run this js, there are small squares showing up against the background until each image renders.  Is that a product of this sample or the the framework?  Perhaps the image should be a:0 until the promise that the image is loaded is received. Not sure if that's what's going on, or if it's something else.  Could you investigate?

**Comment by diatrophy** [comment]
Yes. That is a product of the framework. It looks like the drop shadow is getting rendered prior to the image load. After the image loads the shadow is resized to match the scaled image. I will modify the code so that the shadow (or effect) is a:0 until the image is loaded.

**Comment by diatrophy** [comment]
Should be good now. I have updated the framework to set the initial alpha to 0 (for the effects), and only update it to the appropriate value after the image has been loaded. Please try it out and let me know if it look ok. Thanks.

**Comment by conniefry** [comment]
On the first load of components1.js from browser.js, I am still seeing some little squares show up before the images come in.

Also, please end lines with semi-colon. I know javascript inserts them, but please make it explicit.  Thanks!

**Pull request merged**